### ISSUE TITLE
Auding all sets will report non existent issues

### ISSUE THREAD

**Issue by Toncino** [Robbbert/mameui#61]
Hi, after completing a full Auding of the existing sets, I keep receiving the following missing\missmatch errors, even if Clrmame Pro always reproted that everything is fine:

pool10j     : palce16v8h.u5 (279 bytes) - INCORRECT LENGTH: 2194 bytes
pool10j     : palce20v8h.u22 (343 bytes) - INCORRECT LENGTH: 2706 bytes
pool10j     : palce20v8h.u23 (343 bytes) - INCORRECT LENGTH: 2706 bytes
pool10k     : palce20v8h.u22 (343 bytes) - INCORRECT LENGTH: 2706 bytes
pool10k     : palce20v8h.u23 (343 bytes) - INCORRECT LENGTH: 2706 bytes
sarpc: Romset NOT FOUND

I confirm that the above issues are not real (sarpc starts correctly, for example): any suggestion?

Thanks!

**Comment by Robbbert** [comment]
sarpc doesn't have any roms, that's why you get that message. I don't know about the pool10j issue - you'll need to see if you've got an extra old set in your rompath.

**Comment by Robbbert** [comment]
Nothing heard, closed.

**Comment by Toncino** [comment]
Apologize, I'm currently on vacation and couldn't check the romset on my main computer. I confirm I tried redownloading pool10j and pool10k from different sites, rebuild them with Clrmame Pro and still got the same issue. If I'm not wrong, this didn't happen when I used Split sets, it might have started appearing when I moved to Merged an year ago. Could this be a reason for the issue?
For sarpc, if no rom is needed, why do I receive the error message?
